docs/deepsemgrep/deepsemgrep-introduction.md

```diff
@@ -109,7 +100,7 @@ The following performance differences may arise:
 
 ### Semgrep Pro Engine CI scan issues
 
-To ensure scans reliably complete, Semgrep Pro Engine can fall back to the use of Semgrep OSS Engine. This ensures that in the vast majority of cases, scans run successfully.
+To provide reliability, Semgrep Pro Engine can fall back to the use of Semgrep OSS Engine. This ensures that in the vast majority of cases, scans run successfully.
```

Comment:
The important bit here is that the scan is **completed**. That is more concrete than "reliable" which could mean anything.

Reply:
```suggestion
To provide realibly completed scans, Semgrep Pro Engine can fall back to the use of Semgrep OSS Engine. This ensures that in the vast majority of cases, scans run successfully.
```